src/main/resources/anniv7Resources/localization/eng/CouncilOfGhostsPatron/Characterstrings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "${ModID}:CouncilOfGhostsPatron": {
+    "NAMES": [
+      "First Ghost",
```

Comment:
There isn't anything that relies on this yet, but it might be good to have a first entry here for the collective name. So this would look like:
```
    "NAMES": [
      "Council of Ghosts",
      "First Ghost",
      "Second Ghost",
      "Third Ghost"
    ],
```

This won't affect anything right now, but the reason I'm suggesting this is in case we build something that lists out the name of every patron (e.g. an interface to enable or disable patrons). For that kind of interface we'd want the name of this patron to be the collective "Council of Ghosts" rather than just "First Ghost". You'd need to adjust your code slightly but it wouldn't be too much work.

Regardless, this isn't a big deal, though, just me trying to think about how to future-proof this. Even if you didn't do this it would be easy for us to add it in later!

Reply:
Updated, and updated the cutscene indexes accordingly.